fake_property_data.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
put this file in management folder, like we did in cosmedicos
it will run with this following command
`./manage.py fake_property_data`